string table_id
string[] updated_states
uint8 no_of_people 
uint8 no_of_object
bool need_serving 
bool need_cleaning 
bool require_order 
bool already_served 
bool current_serving 
string[] required_drinks
---
string result 